docs/validators/delegation-manager.mdx

```diff
@@ -136,43 +136,84 @@ Setting the identity of the staking pool in the metadata is the **first step** i
+In order to do so, the owner of the nodes/staking provider must open a Pull Request against https://github.com/multiversx/mx-assets (`master` branch).
 
-In order to complete the matching between the delegation contract and GitHub identity of the staking pool, a repository needs to be created with the name "multiversx" which contains one single file: "keys.json". This file contains a json array with the provider's smart contract address, just like it was on Keybase. For devnet/testnet, you need to have a folder called devnet/testnet in the same repository, with the "keys.json" file underneath.
+_Step 1: choose the environment_
+- for mainnet go to `mx-assets/identities`
+- for testnet go to `mx-assets/testnet/identities`
+- for devnet go to `mx-assets/devnet/identities`
 
-An example for the contents of the `keys.json` file for the `"testmxprovider"` GitHub identity would be:
+_Step 2: create a directory with an unique name_
+Must be lowercase, alphanumeric, with no spaces. For example, `my-new-identity`
 
+_Step 3: add a `info.json` file_
+Create a `info.json` file where you specify the owned address(es), the name, the description and social links.
+Example:
 ```json
- [
-  "erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqr0llllsj732py"
- ]
+{
+  "description": "This is my new identity",
+  "name": "New identity",
+  "website": "http://newidentity.com",
+  "twitter": "https://twitter.com/newidentity",
+  "location": "France",
+  "owners": [
+    "erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqz8llllsh6u4jp", // staking provider address
+    "erd1qyu5wthldzr8wx5c9ucg8kjagg0jfs53s8nr3zpz3hypefsdd8ssycr6th"  // the address of regular validator nodes owner
+  ]
+}
 ```
 
+Inside the owners array:
+- if you want to define an identity of a Staking Provider, add its SC address
+- if you want to define an identity of simple nodes, add the owner address (all the keys owned by that address will have that identity)
+- if you want to define the same identity for both a Staking Provider and simple nodes, set both the address of the Staking Provider and of the owner of the nodes
+
+_Step 4: Open PR and validate the ownership_
+After opening the PR, you also have to validate the ownership. Refer to the --- section
+
+_Done_
+After the PR is merged, the specified information together with the **service fee, percentage filled** and **APR** (for Staking Providers) will be displayed across the ecosystem.
+If this information cannot be found a generic logo and the address is displayed.
+
 An example of how the delegation contract will be displayed based on the information provided in the GitHub is provided below.
 
 ![stakingpool](/img/stakingpool.png)
 
-:::important
-To also connect the validators themselves to a specific GitHub staking pool identity, two additional steps have to be completed:
+:::note
+Before MultiversX Assets, nodes owners needed to update the `Identity` field inside the `prefs.toml` file of each node. This is not required anymore now.
```

Comment:
Maybe you can also add " and you can safely leave the identity empty on your node configuration" to make it more clearer

Reply:
done